outputs/quickwit.go

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package outputs
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+type QuickwitDynamicMapping struct {
+	Description string `json:"description"`
+	Fast        bool   `json:"fast"`
+	ExpendDots  bool   `json:"expand_dots"`
+	Indexed     bool   `json:"indexed"`
+	Record      string `json:"record"`
+	Stored      bool   `json:"stored"`
+	Tokenizer   string `json:"tokenizer"`
+}
+
+type QuickwitFieldMapping struct {
+	Name string `json:"name"`
+	Type string `json:"type"`
+	Fast bool   `json:"fast"`
+}
+
+type QuickwitSearchSettings struct {
+	DefaultSearchFields []string `json:"default_search_fields"`
+}
+
+type QuickwitDocMapping struct {
+	DynamicMapping QuickwitDynamicMapping `json:"dynamic_mapping"`
+	FieldMappings  []QuickwitFieldMapping `json:"field_mappings"`
+	Mode           string                 `json:"mode"`
+	StoreSource    bool                   `json:"store_source"`
+	TimestampField string                 `json:"timestamp_field"`
+}
+
+type QuickwitMappingPayload struct {
+	Id             string                 `json:"index_id"`
+	Version        string                 `json:"version"`
+	SearchSettings QuickwitSearchSettings `json:"search_settings"`
+	DocMapping     QuickwitDocMapping     `json:"doc_mapping"`
+}
+
+func (c *Client) checkQuickwitIndexAlreadyExists(args types.InitClientArgs) bool {
+	config := args.Config.Quickwit
+	if !config.AutoCreateIndex {
+		return false
+	}
+
+	endpointUrl := fmt.Sprintf("%s/%s/indexes/%s/describe", config.HostPort, config.ApiEndpoint, config.Index)
+	quickwitCheckClient, err := InitClient("QuickwitCheckAlreadyExists", endpointUrl, config.MutualTLS, config.CheckCert, args)
+	if nil != err {
+		return false
+	}
+
+	if nil != quickwitCheckClient.sendRequest("GET", "") {
+		return false
+	}
```

Comment:
```suggestion
	if err != nil {
		return false
	}

	if quickwitCheckClient.sendRequest("GET", "") != nil {
		return false
	}
```

to create consistency with the rest of the code base